templates/named.conf.erb

```diff
@@ -0,0 +1,26 @@
+// Generated by Puppet
+// Do not edit manually
+//
+// log to /var/log/named/example.log all events from 
+// info UP in severity (no debug)
+// defaults to use 3 files in rotation
+// BIND 8.x logging MUST COME FIRST in this file
+// BIND 9.x parses the whole file before using the log
+// failure messages up to this point are in (syslog) 
+// typically /var/log/messages
+//
+  logging{
+  channel example_log{
+   file "<%= @log_file %>" versions 3 size 2m;
```

Comment:
This needs a test to ensure that it works and continues to work.

I'll help you write one:

```
context "When given a specific log location" do
  let(:params) {{ :log_dir => '/test_log' }}
  it { should contain_file('..named.conf').with_content(/file "\/test_log\/named\.log" version 3 size 2m/) }
end
```